rs/sns/governance/src/types.rs

```diff
@@ -3494,51 +3882,103 @@ pub(crate) mod tests {
             format!("permissions: [Unspecified, ConfigureDissolveState, ManagePrincipals, SubmitProposal, Vote, Disburse, Split, MergeMaturity, DisburseMaturity, StakeMaturity, ManageVotingPermission, <Invalid permission ({invalid_permission})>]")
         );
     }
-
     #[test]
-    fn test_construct_followees() {
+    fn test_neuron_recipe_construct_followees() {
```

Comment:
From the name of this test I cannot easily understand what exactly is being tested here. 

Could you perhaps explain that in a comment?

Reply:
Done by splitting it up into one test per test case with a descriptive name